src/script/components/app-button.ts

```diff
@@ -1,9 +1,12 @@
 import { LitElement, css, html, customElement, property } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
```

Comment:
link style changes, I'm thinking for sake of reducing complexity and better reuse, just creating a separate component, or basing it off the fast element link component. Let me know your thoughts

Reply:
I have no objections. Use your best judgment.